nntrainer/tensor/manager.h

```diff
@@ -192,8 +192,8 @@ class Manager {
    * @return created tensors list
    */
   std::vector<Var_Grad *>
-  requestInputs(const GraphNode &node,
-                const std::vector<TensorDim> &inputs_dim);
+  requestInputs(const GraphNode &node, const std::vector<TensorDim> &inputs_dim,
```

Comment:
Could you specify the behavior in the doxygen?

Reply:
done.